src/plugins/intel_npu/src/plugin/npuw/util.cpp

```diff
@@ -1121,6 +1311,11 @@ void ov::npuw::util::unpack(const ov::SoPtr<ov::ITensor>& from,
     //     - [32, 1, 4096].*[32, 128, 4096]
     //     - [32, 1, 11008].*[32, 128, 11008]
     //     - [86, 1, 4096].*[86, 128, 4096]
+    // unpack_u4f16_asymm_zp:
+    //      ZPs: [256, 16, 1], [2048, 16, 1], [5632, 16, 1]
```

Comment:
I don't get it. What is what here?

Reply:
These are the shapes of zero points. I will clear the comments.